.coderabbit.yaml

```
@@ -0,0 +1,41 @@
+language: ko-KR
+early_access: false
+#tone_instructions:
+reviews:
+  profile: assertive
+  request_changes_workflow: true
+  high_level_summary: true
+  high_level_summary_placeholder: '@coderabbitai summary'
+  auto_title_placeholder: '@coderabbitai title'
+  poem: false
+  review_status: true
+  collapse_walkthrough: false #접기
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)